pkg/command/service/generate.go

```diff
@@ -188,7 +188,7 @@ kperf service generate —n 500 —-interval 20 —-batch 20 --min-scale 0 --max
 	ksvcGenCommand.Flags().StringVarP(&generateArgs.namespaceRange, "namespace-range", "", "", "")
 	ksvcGenCommand.Flags().StringVarP(&generateArgs.namespace, "namespace", "", "", "Namespace name. The ksvc will be created in the namespace")
 
-	ksvcGenCommand.Flags().StringVarP(&generateArgs.svcPrefix, "svc-prefix", "", "testksvc", "ksvc name prefix. The ksvcs will be svcPrefix1,svcPrefix2,svcPrefix3......")
+	ksvcGenCommand.Flags().StringVarP(&generateArgs.svcPrefix, "svc-prefix", "", "testksvc", "ksvc name prefix. The ksvcs will be svcPrefix-1,svcPrefix-2,svcPrefix-3......")
```

Comment:
@zyjiaobj This will be covered in  https://github.com/knative-sandbox/kperf/pull/23/files, so you can leave it there for now.